src/vs/workbench/parts/feedback/electron-browser/feedbackStatusbarItem.ts

```diff
@@ -18,6 +18,7 @@ import { IWorkspaceContextService, WorkbenchState } from 'vs/platform/workspace/
 class TwitterFeedbackService implements IFeedbackService {
 
 	private static TWITTER_URL: string = 'https://twitter.com/intent/tweet';
+	private static TWITTER_MESSAGE_SIZE: number = 280;
```

Comment:
I think this limit is only 280 for languages other than Chinese, Japanese etc. So I am not sure how to best configure it, we should probably have to check for the language that is used.

Reply:
Would checking `navigator.language` be enough?

Reply:
I think what twitter is doing on their web UI is counting each CJK character as double. Also, they don't show a character limit usually, but rather a circular progress bar. Only when a user approaches the character limit do they start showing the actual characters left.

Reply:
So, what we can do is implement a counter which counts double for every CJK character (would need to lookup via code-points, maybe?)

Reply:
@kumarharsh yes if that is what twitter does when they count it seems fine to do the same for us. I think checking for the code-point is a valid approach. Maybe check https://github.com/Microsoft/vscode/blob/master/src/vs/base/common/strings.ts#L574 as inspiration?

Reply:
@bpasero After [digging some more](https://developer.twitter.com/en/docs/developer-utilities/twitter-text.html), I think the solution is not so straightforward. I think with some of the new changes which twitter made (excluding urls, handles, etc from the message length), it might be better to use their [own service](https://github.com/twitter/twitter-text) for determining the message limit. The v2 version will support the 280 character limit.

Here is the issue tracking the release of v2 of twitter-text: https://github.com/twitter/twitter-text/issues/196. Would it be ok to wait for it?